tests/kernel/wave/wave_e2e_test.py

```diff
@@ -17,37 +17,43 @@ def test_copy():
     N = tkl.sym.N
     BLOCK_M = tkl.sym.BLOCK_M
     BLOCK_N = tkl.sym.BLOCK_N
+    ELEMS_PER_THREAD = tkl.sym.ELEMS_PER_THREAD
     ADDRESS_SPACE = tkl.sym.ADDRESS_SPACE
 
     constraints: list[tkw.Constraint] = [
         tkw.HardwareConstraint(
-            threads_per_wave=64, waves_per_block=(1, 1, 1), vector_shapes={M: 16, N: 16}
+            threads_per_wave=64,
+            waves_per_block=(1, 1, 1),
+            # vector_shapes={M: 1, N: ELEMS_PER_THREAD}, # TODO: Symbol doesn't work here, but should
+            vector_shapes={M: 1, N: 4},
         )
     ]
     constraints += [tkw.WorkgroupConstraint(M, BLOCK_M, 0)]
     constraints += [tkw.WorkgroupConstraint(N, BLOCK_N, 1)]
-    constraints += [tkw.WaveConstraint(M, BLOCK_M)]
-    constraints += [tkw.WaveConstraint(N, BLOCK_N)]
+    constraints += [tkw.WaveConstraint(M, 64)]

```

Comment:
I don't think the WAVE_TILE_SIZE_M > WG_TILE_SIZE_M makes sense. Also to my understanding, `WG_TILE_SIZE_M  / WAVE_TILE_SIZE_M == waves_per_block[0]` .

Reply:
You can revert it, it should still pass. :)

Reply:
Yeah, I remember, something wasn't working with `tkw.WaveConstraint(M, 1)`, but current version works.